src/webextension/background.js

```diff
@@ -4,43 +4,55 @@
  *
  * https://wiki.mozilla.org/Compatibility/Go_Faster_Addon/Override_Policies_and_Workflows#User_Agent_overrides
  */
-const contentScripts = [
-  {
-    matches: ["*://webcompat-addon-testcases.schub.io/*"],
-    css: [{file: "injections/css/bug0000000-dummy-css-injection.css"}],
-    js: [{file: "injections/js/bug0000000-dummy-js-injection.js"}],
-    runAt: "document_start"
-  },
-  {
-    matches: ["https://ib.absa.co.za/*"],
-    js: [{file: "injections/js/bug1452707-window.controllers-shim-ib.absa.co.za.js"}],
-    runAt: "document_start"
-  },
-  {
-    matches: ["http://histography.io/*"],
-    js: [{file: "injections/js/bug1457335-histography.io-ua-change.js"}],
-    runAt: "document_start"
-  },
-  {
-    matches: ["*://*.bankofamerica.com/*"],
-    js: [{file: "injections/js/bug1472075-bankofamerica.com-ua-change.js"}],
-    runAt: "document_start"
-  },
-  {
-    matches: ["http://202.166.205.141/bbvrs/*"],
-    js: [{file: "injections/js/bug1472081-election.gov.np-window.sidebar-shim.js"}],
-    runAt: "document_start",
-    allFrames: true
-  }
-];
+const contentScripts = {
+  universal: [
+    {
+      matches: ["*://webcompat-addon-testcases.schub.io/*"],
+      css: [{file: "injections/css/bug0000000-dummy-css-injection.css"}],
+      js: [{file: "injections/js/bug0000000-dummy-js-injection.js"}],
+      runAt: "document_start"
+    }
+  ],
+  desktop: [
+    {
+      matches: ["https://ib.absa.co.za/*"],
+      js: [{file: "injections/js/bug1452707-window.controllers-shim-ib.absa.co.za.js"}],
+      runAt: "document_start"
+    },
+    {
+      matches: ["http://histography.io/*"],
+      js: [{file: "injections/js/bug1457335-histography.io-ua-change.js"}],
+      runAt: "document_start"
+    },
+    {
+      matches: ["*://*.bankofamerica.com/*"],
+      js: [{file: "injections/js/bug1472075-bankofamerica.com-ua-change.js"}],
+      runAt: "document_start"
+    },
+    {
+      matches: ["http://202.166.205.141/bbvrs/*"],
+      js: [{file: "injections/js/bug1472081-election.gov.np-window.sidebar-shim.js"}],
+      runAt: "document_start",
+      allFrames: true
+    }
+  ],
+  fennec: []
+};
 
 /* globals browser */
 
 let port = browser.runtime.connect();
 let registeredContentScripts = [];
 
-function registerContentScripts() {
-  contentScripts.forEach(async (contentScript) => {
+async function registerContentScripts() {
+  let platform = "desktop";
+  let platformInfo = await browser.runtime.getPlatformInfo();
+  if (platformInfo.os == "android") {
```

Comment:
This is fine, but I'm wondering if we'll need to consider GeckoView separately from Fennec at some later date. Perhaps using "android" instead of "fennec" would make it clearer for future readers that overrides should cover both browsers?

Reply:
Hm, that's a good point for the future, I was thinking about that as well. Right now, I decided to stick with `fennec`, as that is what we are using in `ua_overrides.jsm`. There, I have access to `Services.appinfo.name.toLowerCase()`, which returns `fennec`, and I wanted to keep it consistent.

You are right, though, this might also catch GeckoView products, or any other products on Android, if they ever receive extensions.

Reply:
Renamed it to "android" instead of "fenenc". No-surprises is better than having the same identifier in two places, especially since they technically don't do the same thing. Thanks. :)